docs/iris/src/whatsnew/1.9.rst

```diff
@@ -0,0 +1,115 @@
+What's New in Iris 1.9
+**********************
```

Comment:
I'm afraid I don't know anything much about Sphinx recommended styles.
I simply changed this to be compatible with the 1.8 version :  The original version from aggregate_directory had "==" underlining, but this messed up the whatsnew index page and also the overall contents page.
Each Whatsnew page is in fact *not* a chapter, but a section of the Whatsnew chapter (i.e. ehatsnew/index.rst), according to the contents layout.

So, I don't really get what you're asking for here ?

The Sphinx RST notes you linked seem to imply that all the characters are basically equivalent and overlining is optional, so "the structure is determined from the succession of headings".  So ***functionally*** it simply doesn't matter, no ?

***Stylistically***, I think we have never been entirely consistent here anyway ...
In the main index we have links to various pages whose top headings are formatted like this:
```
****************  
Installing Iris  
****************

===============  
Iris user guide  
===============

============================
Iris reference documentation
============================

((gallery : not RST derived, no actual heading/title text))

#####################
 Iris developer guide
#####################

============================
Iris technical 'Whitepapers'
============================

((what's new : link is to latest section, not the overall "whatsnew index"))
```

The individual Userguide "chapters", at the next level down, also all have headings with === above and below, and so do all the pages in the (auto-generated) reference docs.